geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/DiskStoreCommands.java

```diff
@@ -412,7 +411,7 @@ public Result compactDiskStore(
       @CliOption(key = CliStrings.COMPACT_DISK_STORE__NAME, mandatory = true,
           optionContext = ConverterHint.DISKSTORE,
           help = CliStrings.COMPACT_DISK_STORE__NAME__HELP) String diskStoreName,
-      @CliOption(key = CliStrings.COMPACT_DISK_STORE__GROUP,
+      @CliOption(key = CliStrings.GROUP,
```

Comment:
looks like this can have "groups" as synonym as well. Can you go through all your changes and make sure that member/group CliOption that take String[] as the option type needs to have synonym defined. Thanks!